test/client/session.lua

```diff
@@ -0,0 +1,194 @@
+local safe_pcall
```

Comment:
is `test/client/session.lua` actually only for tests, or (eventually) would we move it to `runtime/...` ?

Reply:
eventually. regardless, `coxpcall` is very small and pure lua, so we could vendor it in `runtime/` when this happens.